Yash Dewangan/templates/index.html

```diff
@@ -0,0 +1,119 @@
+
+    <!--<link rel="stylesheet" href="../static/css/main.css" />-->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css" />
+</head>
+<body>
+    <nav class="navbar" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+    <a class="navbar-item" href="#">
+      <img src="https://bulma.io/images/bulma-logo.png" width="112" height="28">
+    </a>
+
+    <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+    </a>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-start">
+      <a class="navbar-item" href="#">
+        Check Weather
+      </a>
+
+      <a class="navbar-item" href="#">
+      Check weather forecast 
+      </a>
+
+      
+    </div>
+
+    <div class="navbar-end">
+      <div class="navbar-item">
+        <div class="buttons">
+          <p class="control">
+            <a class="button is-success is-inverted " href="#">
+              <span class="icon">
+                <i class="fab fa-github"></i>
+              </span>
+              <span>Download</span>
+            </a>       
+          </p>
+          
+        </div>
+      </div>
+    </div>
+  </div>
+</nav>
+
+<section class="section">
+  <div class="hero-body">
+    <div class="container has-text-centered">
+        <div class="column is-6 is-offset-3">
+            <h1 class="title">
+                Check Weather
+            </h1>
+            <h2 class="subtitle">
+                
+            </h2>
+            <div class="box">
+                <div class="field is-grouped">
+                    <form action="/" method="POST">
+                     
+                      <div class="control">
+                        <input class="input" type="text" name="city" placeholder="Normal input">
+                      </div>
+                      <div class="control">
+                        <button class="button is-primary">Submit</button>
+                      </div>
+                    </form>  
+                </div>
+            </div>
+        </div>
+    </div>
+  </div>
+</section>
+
+
+<section class="section">
+  <div class="container">
+      <div class="columns">
+          <div class="column is-offset-4 is-4">
+              
+              <div class="box" >
+                  <article class="media">
+                      <div class="media-left">
+                          <figure class="image is-50x50">
+                              <img src="http://openweathermap.org/img/w/{{ weather.icon }}.png" alt="Image" >
+                          </figure>
+                      </div>
+                      <div class="media-content">
+                          <div class="content" >
```

Comment:
Nothing Should be shown here when there is no input in submit box and for the very first time when site is open it showing None can you change it and make it appear when any city is submitted?